packages/react-scripts/config/env.js

```diff
@@ -8,10 +8,10 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 // @remove-on-eject-end
+'use-strict';
```

Comment:
Never mind, I figured it out. I'll send another PR because it needs to be fixed for other reasons too.